wallet/connect/index.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Connect to MetaMask using various options.
+---
+
+# Connect to MetaMask
+
+You can connect your dapp to MetaMask using one of the following methods:
+
+- The [MetaMask SDK](metamask-sdk/index.md)
```

Comment:
```suggestion
- [MetaMask SDK](metamask-sdk/index.md)
```

I believe we refer to MetaMask SDK without the "The" (based on [marketing landing page](https://metamask.io/sdk/)). If so, please revert all instances.